components/playlist/browser/resources/components/playlistItem.tsx

```diff
@@ -322,3 +338,20 @@ export default function PlaylistItem ({
     </PlaylistItemContainer>
   )
 }
+
+export function SortablePlaylistItem (props: Props) {
+  const { attributes, listeners, setNodeRef, transform, transition } =
+    useSortable({ id: props.item.id })
```

Comment:
It looks like `useSortable` has a `disabled` argument - I wonder if we could use that instead of conditionally adding the `DndContext` and wrapping the playlist item in a `SortablePlaylistItem`?

Reply:
Yeah, that'd be better. But I think we'll still need wrapping the playlist item in a SortablePlaylistItem for `DragOverlay`. The doc says we should avoid using the same "sortable" component in drag overlay. Because of the id collision, that might lead to unexpected behavior. So I guess we'll need a plain `PlaylistItem` for the `DragOverlay`

Reply:
Did not know that! All right, this looks good then!